packages/beacon-node/src/network/processor/index.ts

```diff
@@ -20,19 +20,24 @@ export type NetworkProcessorOpts = GossipHandlerOpts & {
   maxGossipTopicConcurrency?: number;
 };
 
-const executeGossipWorkOrderObj: Record<GossipType, true> = {
+/**
+ * True if we want to process gossip object immediately, false if we check for bls and regen
+ * in order to process the gossip object.
+ */
+const executeGossipWorkOrderObj: Record<GossipType, boolean> = {
```

Comment:
Make this an Record of enum for better readibility:
```suggestion
type WorkOpts = {
  bypassQueue?: boolean
}

const executeGossipWorkOrderObj: Record<GossipType, WorkOpts> = {
  [GossipType.beacon_block]: {bypassQueue: true}
```